next.config.js

```diff
@@ -98,17 +98,22 @@ module.exports = withSwingset({
 			return globRedirects
 		},
 		async rewrites() {
-			const rewrites = await rewritesConfig()
+			const { variantRewrites, generalRewrites } = await rewritesConfig()
+			await fs.promises.writeFile(
+				path.join('src', 'data', '_variant-rewrites.generated.json'),
+				JSON.stringify(variantRewrites, null, 2),
+				'utf-8'
+			)
```

Comment:
💭 Is this the right place to do this?

Reply:
Whereever we do write it, should add a note about the [edge config ](https://vercel.com/docs/storage/edge-config)